### ISSUE TITLE
Error: HTTP 507 Insufficient Storage

### ISSUE THREAD

**Issue by zacc1111** [mozilla-services/syncserver#182]
Hello my Sync Server do not work.

Ubuntu Server 18.04 LTS on Raspberry Pi

my nginx config:

       location ^~ /fsync/ {
                proxy_set_header Host $http_host;
                proxy_set_header X-Forwarded-Proto $scheme;
                proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
                proxy_set_header X-Real-IP $remote_addr;
                proxy_redirect off;
                proxy_read_timeout 120;
                proxy_connect_timeout 10;
                proxy_pass http://127.0.0.1:5000/;
                proxy_max_temp_file_size 200M;
                client_max_body_size 200M;

        }

error log:
1570653075745	Sync.Engine.Extension-Storage	ERROR	Syncing {73a6fe31-595d-460b-a920-fcc0f8843232}: request failed: Error: HTTP 507 Insufficient Storage: Resource access is forbidden for this user (Maximum bytes per object exceeded " "(17275 > 16384 Bytes.)(resource://services-common/kinto-http-client.js:2771:5) JS Stack trace: ServerResponse@kinto-http-client.js:2771:5
processResponse@kinto-http-client.js:2918:13
**1570653075745	Sync.Engine.Extension-Storage	WARN	Syncing failed: Error: HTTP 507 Insufficient Storage: Resource access is forbidden for this user (Maximum bytes per object exceeded " "(17275 > 16384 Bytes.)**(resource://services-common/kinto-http-client.js:2771:5) JS Stack trace: ServerResponse@kinto-http-client.js:2771:5
processResponse@kinto-http-client.js:2918:13
1570653075746	Sync.Status	DEBUG	Status for engine extension-storage: error.engine.reason.unknown_fail
1570653075746	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed_partial
1570653075746	Sync.ErrorHandler	DEBUG	extension-storage failed: Error: HTTP 507 Insufficient Storage: Resource access is forbidden for this user (Maximum bytes per object exceeded " "(17275 > 16384 Bytes.)(resource://services-common/kinto-http-client.js:2771:5) JS Stack trace: ServerResponse@kinto-http-client.js:2771:5
processResponse@kinto-http-client.js:2918:13

**Comment by rfk** [comment]
Thanks for the report. This looks like webextension storage is trying to upload an object that's too big for the server to accept, @glasserc does that seem accurate?

**Comment by glasserc** [comment]
Yes, that's correct. The webextension storage server isn't covered by this repository, instead it's in https://github.com/Kinto/kinto. Unfortunately we don't have any documentation about how to set it up for self-hosting, sorry :(
